### ISSUE TITLE
Accessibility:MAS36: Not able to navigate to the Menu Bar using Tab key.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17278]
VS Code Version: 1.6.1
OS Version:1607

**Steps to Reproduce:**
1.Open visual studio code editor.
2.Using "Tab", key try to navigate to the "Menu Bar"

**Expected result:**
User should be able to navigate to all the "Menu Bar" using Tab key

**Actual Result:**
User is not able to navigate to the Menu bar options using "Tab" key.

**User Impact:**
User might wouldn't able to Navigate to Menu Bar without using short cut keys

**Attachments:**
![not able to navigate to the menu bar using tab key](https://cloud.githubusercontent.com/assets/22956130/21217759/dd0e0144-c2d3-11e6-9592-63ea5d51ca66.jpg)

**Comment by WiproPSAccTest** [comment]
Title has been changed to a common format to perform data population.

**Comment by isidorn** [comment]
Dependent on https://github.com/electron/electron/issues/2504

**Comment by bpasero** [comment]
https://github.com/Microsoft/vscode/issues/1569
